examples/run_glue.py

```diff
@@ -505,6 +515,16 @@ def main():
 
     logger.info("Training/evaluation parameters %s", args)
 
+    # Start a wandb run
+    if args.log_on_wandb:
+        if not _WANDB_AVAILABLE:
+            logger.error("To use Weights & Biases logging, import wandb.")
+        else:
+            wandb.init(config={**vars(args), 'script': os.path.basename(__file__), 'config':config.to_dict(),
+                               'tokenizer_config':tokenizer.init_kwargs})
+        # keep track of model topology and gradients
+        wandb.watch(model)
```

Comment:
Don't call watch if not `_WANDB_AVAILABLE`

Reply:
Good catch